### PR TITLE
Added EnableWordTimeOffset=true to LongRunningRecognize (Async file)

### PR DESCRIPTION
**Pull request by dlaqab**
Added EnableWordTimeOffset=true to LongRunningRecognize (Async file)
Added EnableWordTimeOffset=false to Sync samples

### CODE REVIEW

**Comment by dlaqab** [comment]
I am covered under my employer (Google)

File: speech/api/Recognize/Recognize.cs [hunk 1]
```diff
@@ -99,6 +99,7 @@ public class Recognize
                 Encoding = encoding,
                 SampleRateHertz = bitRate,
                 LanguageCode = "en",
+                EnableWordTimeOffsets = false,
```
Comment (SurferJeffAtGoogle) [inline speech/api/Recognize/Recognize.cs]: Why show users to set a value when it's the same as the default?

**Comment by remi** [comment]
The EnableWordTimeOffset=false parameter in these samples is extraneous to the task these samples are intended to demonstrate. Closing.

**Comment by remi** [comment]
Note: samples which demonstrate Enable Word Time Offsets were added to dotnet-docs-samples in #310

**Pull request closed without merging**